docs/ui-legacy/auth/fragments/react/sign-up-attributes.md

```diff
@@ -0,0 +1,60 @@
+<table class="doc-table">
```

Comment:
Doesn't look like these tables will get style because this css is out of scope. We should migrate this to use md tables.

Reply:
Correct. They don't get the consistent styling as we get using <ui-component-props>
Do we have a styling in md that would match the ones we have for the new ui library  ?